pkg/refactor/datastore/datastore.go

```diff
@@ -23,7 +24,7 @@ type Store interface {
 
 	// Logs returns logengine.LogStore, is responsible for reading and writing logs
 	Logs() logengine.LogStore
-
+	NewLogs() plattformlogs.LogStore
```

Comment:
@h8718 , All the implementations and interfaces of stores need to be in datastore package.